docs/features.md

```diff
@@ -89,16 +89,42 @@ async def json(request):
     return jsonify({"hello": "world"})
 ```
 
-## Returning a JSON Response
-You can also serve JSON responses when serving HTTP request using the following way.
+## Format of the Response
+Robyn supports several kind of Response for your routes
 
-```python
-from robyn import jsonify
+#### JSON
+You can serve JSON responses when serving HTTP request in the following way.
 
-@app.post("/jsonify")
+```python
+@app.post("/json")
 async def json(request):
     print(request)
-    return jsonify({"hello": "world"})
+    return {
+        "status_code": 200,
+        "body": "This is a regular response",
+        "type": "text",
+        "headers": {"Header": "header_value"},
+    }
```

Comment:
Hey @AntoineRR ,

The return type of json should be 

```python
return {
        "status_code": 200,
        "body":  jsonify({"hello": "world"},
        "headers": {"Content-Type": "application/json"},
    }
```

or just

```
return jsonify({"hello": "world"})
```

It will be returning a Plain Text otherwise

Reply:
Oh my bad you're right, I will change this :)

Reply:
@sansyrox it should be ok now :slightly_smiling_face: